### ISSUE TITLE
Consider adding config file support

### ISSUE THREAD

**Issue by alexhallam** [alexhallam/tv#70]
It would be nice if a `.tv.toml` existed. The keys would match the command line options. Mostly this would be a way for users to specify their color theme. On a linux the program would look for `$HOME/.tv.toml`. I am not sure how dotfiles are setup on other OSes.

This issue is open for discussion.

**Comment by alexhallam** [comment]
I think this will be possible with this create [https://github.com/dirs-dev/directories-rs](https://github.com/dirs-dev/directories-rs)

**Comment by alexhallam** [comment]
Just wanted to put one possible implementation here:

In `main.rs`

```rs
use directories::{BaseDirs};
use std::path::PathBuf;
use serde::{Deserialize};
use toml;

fn main() {

    #[derive(Deserialize, Debug)]
    struct Data {
        name: String,
        color: usize,
        delimiter: String,
        title: String,
        footer: String,
        upper_column_width: usize,
        lower_column_width: usize,
        number: usize,
    }

   if let Some(base_dirs) = BaseDirs::new() {
       let config_dir = base_dirs.config_dir();
       let conf_file = PathBuf::from("tv.toml");
       let conf_dir_file = config_dir.join(conf_file.clone());
       println!("config_dir {:?}",config_dir);
       println!("conf_file {:?}",conf_file);
       println!("conf_dir_file {:?}",conf_dir_file);
   // let file_contents = std::fs::read_to_string("/home/ubuntu/.config/tv.toml").unwrap();
    let file_contents = std::fs::read_to_string(conf_dir_file).unwrap();
    let contents: Data = toml::from_str(&file_contents.to_owned()).unwrap();
    println!("{:#?}", contents);
   }
}
```

in `.config/tv.toml`

```toml
name = "Tidy-Viewer Default Config"
# There are 4 colors (1)nord, (2)one_dark, (3)gruvbox, and (4)dracula. An input of (0)bw will remove color
# properties. Note that colors will make it difficult to pipe output to other utilities [default: 1]
color = 1
# The delimiter separating the columns. [default: ,]
delimiter = ","
# Add a title to your tv. Example 'Test Data' [default: NA ("")]
title = ""
# Add a footer to your tv. Example 'footer info' [default: NA ("")]
footer = ""
# The upper (maxiumum) width of columns. [default: 20]
upper_column_width = 20
# The lower (minimum) width of columns. Must be 2 or larger. [default: 2]
lower_column_width = 2
# number of rows to output <row-display>     Show how many rows to display. [default: 25]
number = 25
```

**Comment by alexhallam** [comment]
Closed with #90
